src/issueModal/sass/issue-modal.scss

```
@@ -264,15 +268,16 @@
-			margin: 16px 0;
-		}
-
+		// Modal-specific layout overrides for fix/dismiss panels.
+		// The component-level styles for .edac-analysis__dismiss-panel and its
+		// children live in _dismiss-panel.scss at root scope.
 		.edac-analysis__fix-panel,
 		.edac-analysis__dismiss-panel {
 
 			.components-panel {
 				border: none;
+				// Negative margin compensates for the modal's 24px side padding
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)